website/config/routes.js

```diff
@@ -392,8 +392,12 @@ module.exports.routes = {
   'GET /handbook/digital-experience': '/handbook/marketing#digital-experience',
   'GET /handbook/digital-experience/article-formatting-guide': '/handbook/marketing/article-formatting-guide',
   'GET /handbook/digital-experience/commonly-used-terms': '/handbook/marketing/commonly-used-terms',
+  'GET /handbook/marketing/commonly-used-terms',: '/handbook/company/communications#commonly-used-terms',
```

Comment:
```suggestion
  'GET /handbook/marketing/commonly-used-terms': '/handbook/company/communications#commonly-used-terms',
```
Can you remove the comma before the colon on this line?